### PR TITLE
[WIP] First draft

### PR DESCRIPTION
**Pull request by eezcjkr**
needs a bunch of minor changes to be complete

### CODE REVIEW

File: setup.js [hunk 1]
```diff
@@ -1,16 +1,112 @@
...
+  const alice = accounts[0].addr;
+  console.log(alice);
+  web3.eth.getBalance(alice).then(console.log);
+  await contracts.token.methods.approve(contracts.exitHandler.options.address, 500000000000).send({from: alice});
+  await contracts.token.methods.approve(contracts.operator.options.address, 500000000000).send({from: alice});
+
+  let slotId = 0;
+  for (let i = 0; i < nodes.length; i++) { 
+    let node = nodes[i];
+    const info = await node.web3.getValidatorInfo();
```
Comment (sunify) [inline setup.js]: This shouldn't be a part of our API, you can fetch it from tendermint directly

Comment (sunify) [reply setup.js]: But if we'll implement https://github.com/leapdao/leap-node/issues/108 it can come in handy 🤔

Comment (eezcjkr) [reply setup.js]: I think this is very useful. It basically allows us to say to the node: please identify yourself. We definitely need this for the ethereum address, so might as well add tendermint next to it no?

File: run.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    token: tokenContract
+  }
+
+  const accounts = getAccounts(config.mnemonic, 5);
+
+  await setup(contracts, nodes, accounts, web3);
+  // Wait for setup to propagate to all the nodes
+  await sleep(15000);
+
+  var testPath = require("path").join(__dirname, "tests");
```
[the remark this answers is not in the document]
Comment (eezcjkr) [reply run.js]: Noted and will be done, but closing this PR for now.

**Pull request merged**